sources/api/migration/migrations/v1.1.0/shared-containerd-configs/src/main.rs

```diff
@@ -0,0 +1,102 @@
+#![deny(rust_2018_idioms)]
+
+use lazy_static::lazy_static;
+use migration_helpers::{migrate, Migration, MigrationData, Result};
+use std::process;
+
+const SETTING: &'static str = "configuration-files.containerd-config-toml.template-path";
+
+lazy_static! {
+    static ref TEMPLATE_CHANGES: &'static [(&'static str, &'static str)] = &[
```

Comment:
This could be a candidate for a plural version of our ReplaceStringMigration. I guess we haven't needed that yet because we haven't changed settings that differ across variants. We could also wait until we need it once more and then use this as a template for making a common migration.